### DISCUSSION TITLE
Mask vs YOLOv3 performance

### DISCUSSION THREAD

**Discussion by RutenburgIG** [open-mmlab/mmdetection#9120] [Q&A] [answered]
Hi folks,
I recently was looking to replace MaskRCNN with YOLOv3 since many resources report at least an x5 throughput bump with the second one. But I see following:
<meta charset="utf-8"><b style="font-weight:normal;" id="docs-internal-guid-e496ac26-7fff-0059-0daa-59c65b63ade0">

  | MaskRCNN | YOLOv3 | delta
-- | -- | -- | --
Infer mean time (sec) | 0.157 | 0.142 | -10%
Infer median time (sec) | 0.160 | 0.149 | -7%

</b>
By infer time I get here model.forward() call. Configurations for both architectures are identical in the input size and batch size.
So instead of x5 speed up, I have only 10% :( Does anyone have some ideas/clues about what's going on and how it could be?

**Comment by ElectronicElephant** [comment, marked as answer]
For YOLOv3, you should try a smaller input size.

To be exact, the default input size of Mask-RCNN is 1333 by 800, while for YOLOv3, it's only 608 by 608~

Definitely, YOLOv3 will become slower when input size is too large, due to its dense structure.
